pages/common/git-changelog.md

```diff
@@ -0,0 +1,33 @@
+# git changelog
+
+> Generate a changelog report from repository commits and tags.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-changelog>.
+
+- Update existing file or creating a new History.md file with pretty formatted output:
```

Comment:
What is a non `pretty` output?
```suggestion
- Update existing file or create a new `History.md` file with the commit messages since the latest Git tag:
```

Reply:
@mfrw can you take a look at this and give me your thoughts as well?